{
  "affected": [
    {
      "database_specific": {
        "source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/ethherium/MAL-2024-10015.json"
      },
      "package": {
        "ecosystem": "PyPI",
        "name": "ethherium",
        "purl": "pkg:pypi/ethherium"
      },
      "versions": [
        "1.0.0"
      ]
    }
  ],
  "credits": [
    {
      "contact": [
        "https://www.reversinglabs.com"
      ],
      "name": "ReversingLabs",
      "type": "FINDER"
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "id": "RLMA-2024-08242",
        "import_time": "2024-10-24T00:56:58.665406953Z",
        "modified_time": "2024-10-16T14:40:49Z",
        "sha256": "fb3db4bdfba973dfad287b06a018e648e26b0a32eff599d3f64b685d452e02a3",
        "source": "reversing-labs",
        "versions": [
          "1.0.0"
        ]
      }
    ]
  },
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "id": "MAL-2024-10015",
  "modified": "2024-10-16T14:40:49Z",
  "published": "2024-10-16T14:40:49Z",
  "schema_version": "1.7.0",
  "summary": "Malicious code in ethherium (PyPI)"
}